### PR TITLE
feat!: combine helm charts

### PR DESCRIPTION
**Pull request by olevski**
/deploy #persist renku=000-combine-test-fix extra-values=initDb.image.repository=registry.dev.renku.ch/tasko.olevski/renku-images/init-db,initDb.image.tag=0.36.4-8.21cbfb5,keycloakx.initRealm.image.repository=registry.dev.renku.ch/tasko.olevski/renku-images/init-realm,keycloakx.initRealm.image.tag=0.36.4-8.21cbfb5

### CODE REVIEW

File: helm-chart/renku-ui/values.yaml [hunk 1]
```diff
@@ -1,299 +1,10 @@
...
-  securityContext:
-    runAsUser: 1000
-    runAsGroup: 1000
-    runAsNonRoot: true
-    allowPrivilegeEscalation: false
-
-  service:
-    type: ClusterIP
-    port: 80
+      tag: "3.12.2"
```
Comment (lorenzo-cavazzi) [inline helm-chart/renku-ui/values.yaml:5]:
```suggestion
      tag: "3.13.0"
```

File: helm-chart/renku-ui/values.yaml [hunk 2]
```diff
@@ -1,299 +1,10 @@
...
-  prometheus:
-    enabled: true
-    annotations:
-      prometheus.io/path: "/metrics"
-      prometheus.io/port: "8080"
-      prometheus.io/scrape: "true"
+  server:
+    image:
+      repository: renku/renku-ui-server
+      tag: "3.12.2"
```
Comment (lorenzo-cavazzi) [inline helm-chart/renku-ui/values.yaml:10]:
```suggestion
      tag: "3.13.0"
```

**Review by leafty — APPROVED** [review]
Code changes look good to me 🚀

**Review by lorenzo-cavazzi — COMMENTED** [review]
Perhaps it's just a missing `extraValue` in the `/deploy` string, but currently the deployment doesn't support persistent sessions and tests fail (bottom right in the screenshot)

image

**Comment by olevski** [comment]
> Perhaps it's just a missing extraValue in the /deploy string, but currently the deployment doesn't support persistent sessions and tests fail (bottom right in the screenshot)

Cool I fixed that in the new values file. Thanks Lorenzo.

**Comment by lorenzo-cavazzi** [comment]
@olevski I noticed dataset tests started failing with the combined helm chart. I have no idea why that would be an issue for the tests, but I made [a tiny PR to stabilize them](https://github.com/SwissDataScienceCenter/renku/pull/3277). Tests are re-running now

**Comment by olevski** [comment]
The failing sessions test is because of a bug we introduced in the notebooks with the hibernating sessions features. It is being fixed.

**Pull request merged**